src/wayland/generated/wayland_wrapper.h

```diff
@@ -13,16 +13,17 @@
 #include "mir/fd.h"
 #include <wayland-server-core.h>
 
+#include "wayland_base.h"
```

Comment:
Having generated code dependent on a header we don't publish (src/wayland/base/wayland_base.h) is going to make things difficult for bespoke Wayland protocol writers.

Reply:
Done